src/systems/filecoin_mining/storage_mining/storage_miner_actor_code.go

```diff
@@ -18,6 +18,7 @@ import (
 
 const (
 	Method_StorageMinerActor_SubmitSurprisePoSt = actor.MethodPlaceholder + iota
```

Comment:
```suggestion
	Method_StorageMinerActor_SubmitVerifiedSurprisePoSt = actor.MethodPlaceholder + iota
```

Note that we may want to change it, but current assumption is that the verifier will use the subsystem verification methods ahead of submitting the messages. We could do all in one go.

Reply:
Done (this is unused anyway).